File: app/src/main/java/com/example/vivic/nolost/bean/MyUser.java
package com.example.vivic.nolost.bean;

import cn.bmob.v3.BmobUser;

public class MyUser extends BmobUser {


    public String gender;
    public String location;
    public String avatar;
    public String contact;
    public String background;

    @Override
    public String toString() {
        return " MyUser:" +
                ", Username:" + getUsername() +
                ", gender:" + gender +
                ", location:" + location +
                ", avatar:" + avatar +
                ", background:" + background +
                ", contact:" + contact;
    }


}

File: app/src/main/java/com/example/vivic/nolost/bean/MyUser.kt
package com.example.vivic.nolost.bean

import cn.bmob.v3.BmobUser

class MyUser : BmobUser() {

    var platform: String? = null
    var gender: String? = null
    var location: String? = null
    var avatar: String? = null
    var contact: String? = null
    var background: String? = null

    override fun toString(): String {
        return " MyUser:" +
                ", platform:" + platform +
                ", Username:" + username +
                ", gender:" + gender +
                ", location:" + location +
                ", avatar:" + avatar +
                ", background:" + background +
                ", contact:" + contact
    }

}
